packages/compiler/src/formatter/print/printer.ts

```diff
@@ -1335,6 +1338,21 @@ export function printTypeReference(
   return [type, template];
 }
 
+export function printTemplateArgument(
+  path: AstPath<TemplateArgumentNode>,
+  _options: TypeSpecPrettierOptions,
+  print: PrettierChildPrint
+): Doc {
+  if (path.getNode()?.name !== undefined) {
```

Comment:
```suggestion
  if (path.getValue().name !== undefined) {
```

I think that's the correct way to do it that works for prettier 2.0 and 3.0, at 3.0 we should replace with `.node` but that would break people that didn't update